pkg/talos/talos_machine_configuration_data_source.go

```diff
@@ -172,11 +172,11 @@ func (d *talosMachineConfigurationDataSource) Schema(_ context.Context, _ dataso
 			},
 			"docs": schema.BoolAttribute{
 				Description: "Whether to generate documentation for the generated configuration",
-				Optional:    true,
+				Optional:    false,
 			},
 			"examples": schema.BoolAttribute{
 				Description: "Whether to generate examples for the generated configuration",
-				Optional:    true,
```

Comment:
I don't think we should change `optional`. 

Isn't the default `false` already?

Reply:
default is `true`, if we want to be false I can change that

Reply:
optional should stay as is